includes/functions/functions_files.php

```diff
@@ -144,6 +144,29 @@ function zen_get_module_sidebox_directory($check_file)
     return $template_dir_select . $zv_filename;
 }
 
+/**
+ * find module directory for admin product-type modules
+ * @param string $product_type
+ * @param string $filename_to_check
+ * @param bool $dir_only
+ * @return string
+ */
+function zen_get_admin_module_from_directory($product_type, $filename_to_check, $dir_only = false)
```

Comment:
how would you feel about starting to add types to the signature?  ie:

```
function zen_get_admin_module_from_directory(string $product_type, string $filename_to_check, bool $dir_only = false): string
```

Reply:
Good catch. I was cloning from other functions in the same file. I won't touch those in this PR though, to keep this canonical.